examples/fungible-token/tests/benchmarks.rs

```diff
@@ -0,0 +1,148 @@
+use gclient::{EventProcessor, GearApi, Result, WSAddress};
+use gstd::{vec, Encode, Vec, ActorId};
+use fungible_token::{WASM_BINARY};
+use ft_io::*;
+
+/// This constant defines the number of messages in the batch.
+/// It is calculated empirically, and 25 is considered the optimal value for
+/// messages in this test. If the value were larger, transactions would
+/// exhaust the block limits.
+const BATCH_CHUNK_SIZE: usize = 25;
+const MAX_GAS_LIMIT: u64 = 250_000_000_000;
+
+/// This test runs stress-loading for the fungible token contract.
+/// Its primary purpose is to benchmark memory allocator gas consumption.
+/// It does not verify whether the contract or the runtime works correctly.
+///
+/// See [galloc optimization doc](../../../galloc/docs/optimization.md) for
+/// reference.
+#[ignore = "reason: this test requires a running node, so it is ignored by default"]
```

Comment:
Wouldn't we like to run this test on CI?
Furthermore, there is api to up node by path to binary - better to use it (even for CI purposes)